curator-framework/src/main/java/org/apache/curator/framework/api/transaction/TransactionCreateBuilder.java

```diff
@@ -18,17 +18,17 @@
  */
 package org.apache.curator.framework.api.transaction;
 
-import org.apache.curator.framework.api.ACLCreateModePathAndBytesable;
-import org.apache.curator.framework.api.ACLPathAndBytesable;
-import org.apache.curator.framework.api.Compressible;
-import org.apache.curator.framework.api.CreateModable;
-import org.apache.curator.framework.api.PathAndBytesable;
-
-public interface TransactionCreateBuilder<T> extends
-    PathAndBytesable<T>,
-    CreateModable<ACLPathAndBytesable<T>>,
-    ACLPathAndBytesable<T>,
-    ACLCreateModePathAndBytesable<T>,
-    Compressible<ACLCreateModePathAndBytesable<T>>
+public interface TransactionCreateBuilder<T> extends TransactionCreateBuilder2<T>
```

Comment:
BTW - I think the original Curator DSL was good for its day but the new Async DSL is way cooler :D and easier to understand (I think).